<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>691</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="layout">
       <item row="0" column="2">
        <layout class="QGridLayout" name="InProgressGrid">
         <item row="0" column="0">
          <widget class="QLabel" name="inProgressLabel">
           <property name="text">
            <string>In Progress</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="inProgressListWidget"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="todoGrid">
         <item row="1" column="0">
          <widget class="QListWidget" name="todoListWidget"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="todoLabel">
           <property name="text">
            <string>Todo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QGridLayout" name="InProgressGrid_2">
         <item row="0" column="0">
          <widget class="QLabel" name="inProgressLabel_2">
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="doneListWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="actionButtonsGrid">
       <item row="0" column="0">
        <widget class="QPushButton" name="addTaskButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="deleteTaskButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="editTaskButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="appLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>TODOs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
